custom_components/opnsense/coordinator.py

```
@@ -71,225 +72,231 @@
+                        "state_key": "arp_table",
+                    },
+                ]
+                self._state.update(await self._get_states(categories))
+                if self._state.get("device_unique_id") is None:
+                    _LOGGER.warning(
+                        "Coordinator failed to confirm OPNsense Router Unique ID. Will retry"
+                    )
+                    return {}
+                if self._state.get("device_unique_id") != self._device_unique_id:
+                    _LOGGER.error(
+                        "Coordinator error. OPNsense Router Device ID (%s) differs from the one saved in hass-opnsense (%s)",
```

Pylint (3.3.7)

[convention] Line too long (125/100)

(C0301)